pymtl3/stdlib/test/test_utils.py

```diff
@@ -67,3 +80,190 @@ def run_test( self ):
       self.verify_outputs_func( self.model, test_vector )
 
       self.model.tick()
+
+def run_sim( model, dump_vcd=None, test_verilog=False, max_cycles=5000 ):
+
+  # Setup the model
+
+  if dump_vcd:
+    model.config_tracing = TracingConfigs( tracing='vcd', vcd_file_name=dump_vcd )
+
+  model.elaborate()
+
+  if test_verilog:
+    model = TranslationImportPass()( model )
+
+  # Create a simulator
+
+  model.apply( SimulationPass() )
+
+  # Reset model
+
+  model.sim_reset()
+  print()
+
+  # Run simulation
+
+  ncycles = 0
+  while not model.done() and ncycles < max_cycles:
+    print( f"{ncycles:3}: {model.line_trace()}" )
```

Comment:
i think we should add line trace as a option too. Also maybe we can let the user select which backend to use? instead of `test_verilog` we can make the argument `translation_backend` and default to None

Reply:
hmmm

Reply:
ok, I can add the line trace function, but the test_verilog has a different meaning here ...

Reply:
Basically test_verilog is a string option that selects verilator initialization values

Reply:
i see, maybe another argument to select the backend then?

Reply:
not important now